geomesa-accumulo/geomesa-accumulo-datastore/src/test/scala/org/locationtech/geomesa/accumulo/data/stats/usage/QueryStatTransformTest.scala

```diff
@@ -8,46 +8,42 @@
 
 package org.locationtech.geomesa.accumulo.data.stats.usage
 
-import org.apache.accumulo.core.client.mock.MockInstance
-import org.apache.accumulo.core.client.security.tokens.PasswordToken
 import org.apache.accumulo.core.security.Authorizations
 import org.geotools.data.Query
 import org.geotools.filter.text.cql2.CQL
 import org.geotools.geometry.jts.ReferencedEnvelope
 import org.junit.runner.RunWith
+import org.locationtech.geomesa.accumulo.TestWithDataStore
 import org.locationtech.geomesa.accumulo.audit.{AccumuloAuditService, AccumuloQueryEventTransform}
 import org.locationtech.geomesa.accumulo.util.GeoMesaBatchWriterConfig
 import org.locationtech.geomesa.index.audit.QueryEvent
 import org.locationtech.geomesa.index.conf.QueryHints
 import org.locationtech.geomesa.index.geoserver.ViewParams
-import org.specs2.mutable.Specification
 import org.specs2.runner.JUnitRunner
 
 import scala.collection.JavaConverters._
 
 @RunWith(classOf[JUnitRunner])
-class QueryStatTransformTest extends Specification {
+class QueryStatTransformTest extends TestWithDataStore {
 
-  val table = "QueryStatTransformTest"
   val featureName = "stat-writer-test"
 
-  val connector = new MockInstance().getConnector("user", new PasswordToken("password"))
-  connector.tableOperations().create(table)
-
   "QueryStatTransform" should {
 
     "convert query stats to and from accumulo" in {
 
+      ds.connector.tableOperations().create(catalog)
```

Comment:
I know this is just a refactor... but why is this test creating a table manually?!  Maybe enter a ticket to clean this up?

Reply:
usually the stats table is created by the data store, but since we're just testing the reader/writer directly we have to create the table ourselves